src/UI/Implementation/Render/ilJavaScriptBinding.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/* Copyright (c) 2016 Richard Klees <richard.klees@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Implementation\Render;
+
+/**
+ * Wraps global ilTemplate to provide JavaScriptBinding. 
+ */
+class ilJavaScriptBinding implements JavaScriptBinding {
+	/**
+	 * @var \ilTemplate
+	 */
+	private	$global_tpl;
+
+	/**
+	 * @var	integer
+	 */
+	private $id_count = 0;
```

Comment:
These kinds of bugs would scare the hell out of me: very hard to reproduce and almost completely undefined behaviour.

I would stress this: Please make this **extremely** unprobable. :-)